Microsoft.Quantum.Katas/Utils.qs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+//////////////////////////////////////////////////////////////////////
+// This file contains some Q# intrinsic operations
+// needed for the validation of some Katas' answers.
+//////////////////////////////////////////////////////////////////////
+
+namespace Quantum.Kata.Utils {
+    
+    /// # Summary
+    /// Returns how many times a given oracle is executed.
+    function GetOracleCallsCount<'T> (oracle : 'T) : Int { body intrinsic; }
```

Comment:
I think this needs to be an operation, not a function, since its output can depend on global state (that is, it's not deterministic).
```suggestion
     /// # Input
     /// ## oracle
     /// The operation whose call count is being requested.
    operation GetOracleCallsCount<'T> (oracle : 'T) : Int { body intrinsic; }
```